pulpcore/cli/core/artifact.py

```diff
@@ -36,38 +35,13 @@ def artifact(ctx: click.Context, pulp_ctx: PulpContext) -> None:
 @artifact.command()
 @click.option("--file", type=click.File("rb"), required=True)
 @click.option(
-    "--chunk-size", default=1000000, type=int, help="Chunk size in bytes (default is 1 MB)"
+    "--chunk-size", default=1048576, type=int, help="Chunk size in bytes (default is 1 MB)"
```

Comment:
The default web request size limit for nginx is 1 MiB so I’d suggest we keep the default as 1 MB.

Reply:
We could change the help text to say 1000 bytes.

Reply:
Because it's smaller? I'll never understand those data size units...
But i'm ok with either.

Reply:
https://xkcd.com/394/